### ISSUE TITLE
Make multi platform

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/light-hotspot-software#44]
Ensure this works on our light hotspot hardware but also on all other hardware - Nebra RasPi, Nebra ROCK Pi, RAK, Sensecap, pisces etc

**Comment by pritamghanghas** [comment]
@shawaj won't it be helpful to take new branches of all our code and add/remove functionality. Trying to make the same code base for full and light miner will require significant work on abstractions and those will be short lived. We plan to move 100% to light miners.

I might be totally off tangent here. You might be talking about testing the new software stack on all hardware variants.

**Comment by shawaj** [comment]
Yes exactly @pritamghanghas we need to test this works on all hardware because in the future full hotspots will use this software
